AllReadyApp/Web-App/AllReady/wwwroot/css/site.css

```diff
@@ -115,6 +115,26 @@ ul.nav a:focus {
     text-decoration: underline;
 }
 
+ul.nav.nav-tabs > li > a:hover 
+{
+    /* issue 551 */
+    color: #000;
+}

```

Comment:
The `:focus` pseudo-selector also needs to be applied here.
![image](https://cloud.githubusercontent.com/assets/775229/13204382/7d86d716-d894-11e5-8c65-c11910593d55.png)